regexes/device/mobiles.yml

```diff
@@ -8189,7 +8196,7 @@ Lark:
       model: 'FreeMe X4 9'
     - regex: '(?:Lark[ _-])?Evolution[ _](X2 7 3G-GPS|X4[_ ]10\.1|10|X4[_ ]7[ _]IPS)'
       model: 'Evolution $1'
-    - regex: 'Ultimate (X4 10\.1 3G IPS|X4 8s 3G|7i)'
+    - regex: 'Ul[ti]{2}mate (X4 10\.1 3G IPS|X4 8s 3G|7i)'
```

Comment:
This would actually catch `Ultimate`, `Ulttmate`, `Ulitmate` and `Uliimate`, is that on purpose?

Reply:
fixed